severless.go

```diff
@@ -0,0 +1,130 @@
+package yomo
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/yomorun/yomo/internal/core"
+	"github.com/yomorun/yomo/internal/frame"
+	"github.com/yomorun/yomo/logger"
+	"github.com/yomorun/yomo/zipper/tracing"
+)
+
+const (
+	streamFunctionLogPrefix = "\033[31m[yomo:sfn]\033[0m "
```

Comment:
changed from `StreamFunctionLogPrefix` to `streamFunctionLogPrefix`, hide to end user

Reply:
This is temporary, and the Logger needs to be refactored later.